tests/e2e/actions.go

```diff
@@ -250,6 +258,121 @@ func (tr Chain) submitTextProposal(
 	tr.waitBlocks(action.Chain, 1, 10*time.Second)
 }
 
+type UpdateConsumerChainAction struct {
+	Chain               ChainID
+	From                ValidatorID
+	ConsumerChain       ChainID
+	NewOwner            string
+	InitialHeight       clienttypes.Height
+	SpawnTime           uint
```

Comment:
The name is confusing imo - maybe could name it something like "TimeUntilSpawn" or something like that, because afaict this is actually a delay on when it will spawn, starting now

Reply:
Also,either put in the name that this is in milliseconds, or just change it to a time.Duration type

Reply:
this is consistent with all SpawnTime usage in e2e. 
I'll add it to issue [#2198](https://github.com/cosmos/interchain-security/issues/2198) to change it consistently for all occurencies in e2e.